--- a/src/config.hpp
+++ b/src/config.hpp
@@ -1,6 +1,7 @@
 #ifndef STEAM_AUDIO_CONFIG_H
 #define STEAM_AUDIO_CONFIG_H
 
+#include <vector>
 #include "godot_cpp/classes/node3d.hpp"
 #include "godot_cpp/classes/wrapped.hpp"
 #include "phonon.h"
--- a/src/geometry.hpp
+++ b/src/geometry.hpp
@@ -1,6 +1,7 @@
 #ifndef STEAM_AUDIO_GEOMETRY_H
 #define STEAM_AUDIO_GEOMETRY_H
 
+#include <vector>
 #include "godot_cpp/classes/node3d.hpp"
 #include "godot_cpp/classes/wrapped.hpp"
 #include "material.hpp"
--- a/src/geometry_dynamic.hpp
+++ b/src/geometry_dynamic.hpp
@@ -1,6 +1,7 @@
 #ifndef STEAM_AUDIO_DYNAMIC_GEOMETRY_H
 #define STEAM_AUDIO_DYNAMIC_GEOMETRY_H
 
+#include <vector>
 #include "godot_cpp/classes/node3d.hpp"
 #include "material.hpp"
 #include "phonon.h"
--- a/src/server.hpp
+++ b/src/server.hpp
@@ -1,6 +1,7 @@
 #ifndef STEAM_AUDIO_SERVER_H
 #define STEAM_AUDIO_SERVER_H
 
+#include <vector>
 #include "godot_cpp/classes/object.hpp"
 #include "godot_cpp/classes/thread.hpp"
 #include "listener.hpp"
